pages/documentation/grid/model/index.md

```diff
@@ -34,15 +34,17 @@ In the following sections the different network components are described in term
 | $$Aliases$$ | Additional unique identifiers associated with each network component |
 | $$Properties$$ | To add additional data items to network components |
 
-All equipment and the network itself are identified by a unique identifier which is the only required attribute. They can have a human readable name. offer the possibility of adding additional unique identifiers to each component. An alias can be qualified to indicate what it corresponds to.
+All equipment and the network itself are identified by a unique identifier which is the only required attribute. They can have a human-readable name. offer the possibility of adding additional unique identifiers to each component. An alias can be qualified to indicate what it corresponds to.
 
 Properties allow associating additional arbitrary data items under the general schema of pairs `<Key, Value>`.
 
 To identify non-physical network components, one can use the fictitious property that is set to `false` by default.
 
+A network can contain several subnetworks.
+
 ### Validation level
 
-<span style="color: red">TODO</span>
+The validation level can be `EQUIPMENT` or `STEADY_STATE_HYPOTHESIS`. A network at equipment level is a network with missing steady-state hypothesis. This occurs just after SCADA systems, before any state estimation. Once all steady state hypothesis are filled, meaning that a load flow engine has all the data needed to perform a computation, the level switches to `STEADY_STATE_HYPOTHESIS`. For some processes, a minimal validation level of the network is required.
```

Comment:
1. "A network at equipment level is a network with missing steady-state hypothesis"
- If there are several steady-state hypotheses, then it's "hypothes**e**s" (the plural is irregular)
- Id there is only one hypothesis, then it's "with **a** missing steady-state hypothesis"

2. "Once, all steady-state hypothes**e**s... perfom a computation, the **validation** level..."